sys/net/gnrc/pktbuf/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_PKTBUF
+    bool "Configure GNRC_PKTBUF"
+    depends on MODULE_GNRC_PKTBUF
+    help
+        Configure the GNRC_PKTBUF using Kconfig.
+
+if KCONFIG_MODULE_GNRC_PKTBUF
+
+config GNRC_PKTBUF_SIZE
+    int "Maximum size of the static packet buffer"
```

Comment:
Maybe this should rather be an option for `if KCONFIG_MODULE_GNRC_PKTBUF_STATIC` then.

Reply:
Other modules utilizing `GNRC_PKTBUF_SIZE`—if there ever will be some—could be or'd into the if above. For `gnrc_pktbuf_malloc`, `GNRC_PKTBUF_SIZE` is not used and doesn't make sense either, so this is why I think this should be clarified in the menu config.

Reply:
Done. Thanks for the comment